docs/guides/connecting_to_your_data/database/mysql.md

```diff
@@ -1,4 +1,161 @@
+  ]}>
+  <TabItem value="yaml">
+
+Put your connection string in this template:
+
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py#L20-L34
+```
+Run this code to test your configuration.
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py#L45
+```
+
+</TabItem>
+<TabItem value="python">
+
+Put your connection string in this template:
+
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py#L19-L36
+```
+Run this code to test your configuration.
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py#L42
+```
+
+</TabItem>
+</Tabs>
+
+You will see your database tables listed as `Available data_asset_names` in the output of `test_yaml_config()`.
+
+Feel free to adjust your configuration and re-run `test_yaml_config()` as needed.
+
+### 6. Save the Datasource configuration to your DataContext
+
+Save the configuration into your `DataContext` by using the `add_datasource()` function.
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='yaml'
+  values={[
+  {label: 'YAML', value:'yaml'},
+  {label: 'python', value:'python'},
+  ]}>
+  <TabItem value="yaml">
+
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py#L45
+```
+
+</TabItem>
+<TabItem value="python">
+
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py#L45
+```
+
+</TabItem>
+</Tabs>
+
+### 7. Test your new Datasource
+
+Verify your new Datasource by loading data from it into a `Validator` using a `BatchRequest`.
+
+<Tabs
+  defaultValue='runtime_batch_request'
+  values={[
+  {label: 'Using a SQL query', value:'runtime_batch_request'},
+  {label: 'Using a table name', value:'batch_request'},
+  ]}>
+  <TabItem value="runtime_batch_request">
+
+Here is an example of loading data by specifying a SQL query.
+
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py#L48-L61
+```
+
+  </TabItem>
+
+  <TabItem value="batch_request">
+
+Here is an example of loading data by specifying an existing table name.
+
+```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py#L67-L78
+```
+
+  </TabItem>
+</Tabs>
+
+<Congratulations />
+
+## Additional Notes
+
+To view the full scripts used in this page, see them on GitHub:
+
+- [mysql_yaml_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py)
+- [mysql_python_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py)
+
+## Next Steps
+
+<NextSteps />
```

Comment:
The first link `How to create and edit Expectations with instant feedback from a sample Batch of data` was not rendering correctly on the `deploy-preview-2070` build for me... are you seeing something similar on your end?

Reply:
Funny - I noticed that on a different branch I was on earlier. I'll check it out.

Reply:
fixed